enhancedTable/templates.ts

```diff
@@ -73,23 +72,30 @@ export const MENU_TEMPLATE = `
             <md-menu-item type="checkbox" ng-model="ctrl.showFilter" ng-click="ctrl.toggleFilters()" rv-right-icon="community:filter">
                 {{ 't.menu.filter.show' | translate }}
             </md-menu-item>
-            <md-menu-divider></md-menu-divider>
-            <md-menu-item>
-                <md-button ng-click="ctrl.print()">
-                    <md-icon md-svg-icon="action:print"></md-icon>
-                    {{ 't.menu.print' | translate }}
-                </md-button>
-            </md-menu-item>
-            <md-menu-item>
-                <md-button ng-click="ctrl.export()">
-                    <md-icon md-svg-icon="editor:insert_drive_file"></md-icon>
-                    {{ 't.menu.export' | translate }}
-                </md-button>
-            </md-menu-item>
-        </md-menu-content>
-    </md-menu>
-</md-menu-bar>
-`;
+            <md-menu-divider></md-menu-divider>`;
+
+    if (printEnabled) {
```

Comment:
Would it be worthwhile using an [`ng-if`](https://docs.angularjs.org/api/ng/directive/ngIf) here? Might make the template a little easier to parse.